src/soundvectors.py

```diff
@@ -541,13 +747,17 @@ def __getitem__(self, sound: typing.Union[str, Sound]) -> FeatureBundle:
             # extract secondary features for second segment
             to_sound_features = [f for df in sound_to_sound.split() for f in df.split("-and-")]
             _, to_secondary_features = self._get_features(to_sound_features)
-            for f in to_secondary_features:
-                if (self.feature_values.get(f, {"domain": ""})["domain"] not in
-                        ["raising", "relative_articulation", "rounding"]):
-                    secondary_features.append(f)
```

Comment:
I think we don't need this handling of features that are not found in `self.feature_values`, because they won't contribute anything to the vectors anyway.